roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +51,12 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    def is_active(self):  # pylint: disable=unused-argument,no-self-use
```

Comment:
For some reason I thought it would be a problem to make this a static method and then override it as an instance method. Apparently it's fine.